newIDE/app/src/BehaviorsEditor/Editors/BehaviorPropertiesEditor.js

```diff
@@ -5,49 +5,93 @@ import * as React from 'react';
 import PropertiesEditor from '../../PropertiesEditor';
 import propertiesMapToSchema from '../../PropertiesEditor/PropertiesMapToSchema';
 import EmptyMessage from '../../UI/EmptyMessage';
-import { Column } from '../../UI/Grid';
+import { Column, Line } from '../../UI/Grid';
 import { type BehaviorEditorProps } from './BehaviorEditorProps.flow';
+import FlatButton from '../../UI/FlatButton';
+import { ColumnStackLayout } from '../../UI/Layout';
 
 type Props = BehaviorEditorProps;
 
-export default class BehaviorPropertiesEditor extends React.Component<Props> {
-  render() {
-    const {
-      project,
-      behavior,
-      object,
-      onBehaviorUpdated,
-      resourceManagementProps,
-    } = this.props;
+const BehaviorPropertiesEditor = ({
```

Comment:
The button is in the `BehaviorPropertiesEditor`. We would need to duplicate the system for objects, but I think it's better not to generalize.

Reply:
Indeed, this might change rapidly.